### PR TITLE
Remove Travis and migrate to Github Actions

### PR DESCRIPTION
**Pull request by rafaeltoledo**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+      - name: Gradle Wrapper Validation
+        uses: gradle/wrapper-validation-action@v1.0.3
+      - name: Install JDK
```
Comment (uKetki) [inline .github/workflows/build.yml:13]: You may need to add the `Install Android SDK` step as follows:
```
      - name: Install Android SDK
        uses: malinskiy/action-android/install-sdk@release/0.1.1
```

**Comment by uKetki** [comment]
You can safely remove the `.buildscript/deploy_snapshot.sh` file now!

**Comment by tyvsmith** [comment]
I suspect it's because it needs to be landed on master prior to PRs being run. Looking at the settings, that screenshoted option doesn't exist, probably because that's for a private repo?

I suggest we merge this in, then verify.

**Comment by tyvsmith** [comment]
@rafaeltoledo @uKetki If one of y'all gets to look at this failure and fix now that we have initial runs, let me know.

**Pull request merged**